contrib/compose/docker-compose-dev.yml

```diff
@@ -87,6 +89,8 @@ services:
     build:
       context: ../..
       dockerfile: ./contrib/images/connect.sidecar.dev.Dockerfile
+    environment:
+    # Put ALPACA_API_KEY and ALPACA_API_SECRET here
```

Comment:
does this run? afaik if you put a key in YAML it requires at least [] to specify it is empty

Reply:
Good catch, fixed that!